Assets/Scripts/BotScript.cs

```diff
@@ -192,5 +226,49 @@ private bool BetweenOne(float value)
 		return value<=1 && value >= -1;
 	}
 
-   
+    Player MapColliderToPlayer(GameObject gameObject)
+    {
+        
+
+        Player player = new();
```

Comment:
On a real project, try to use immutables. Will allow for this project.

Reply:
noted.